Network/HTTP/ReverseProxy.hs

```diff
@@ -353,18 +354,25 @@ waiProxyToSettings getDest wps' manager req0 sendResponse = do
     case edest of
         Left app -> maybe id timeBound (lpsTimeBound lps) $ app req0 sendResponse
         Right (ProxyDest host port, req, secure) -> tryWebSockets wps host port req sendResponse $ do
-            let req' = def
-                    { HC.method = WAI.requestMethod req
+            let req' =
+#if MIN_VERSION_http_client(0, 5, 0)
+                  HC.defaultRequest
+                    { HC.checkResponse = \_ _ -> return ()
+                    , HC.responseTimeout = maybe HC.responseTimeoutNone (HC.responseTimeoutMicro . (1000 *)) $ lpsTimeBound lps
```

Comment:
Yes, I was wondering that :) but `fiveMinutes` is in milliseconds (5 * 60 * 1000) i.e. 1/1000; microseconds are  1/1000000.
I was not checking the underlying implementation; just the name.